Makefile

```diff
@@ -196,7 +196,7 @@ pseudoxml:
 all-source:
 
 locale/circuitpython.pot: all-source
-	find $(TRANSLATE_SOURCES) -iname "*.c" | xargs xgettext -L C -s --add-location=file --keyword=translate -o circuitpython.pot -p locale
+	find $(TRANSLATE_SOURCES) -iname "*.c" -print0 | sort -z | xargs xgettext -L C -s --add-location=file --keyword=translate -o circuitpython.pot -p locale
```

Comment:
I don't think the ```-print0``` and ```-z``` options are needed but if you do use them I think you also need ```-0``` for the xargs command.

Reply:
Done